SeleniumTest/test-output/Default suite/Default test.html

```diff
@@ -0,0 +1,84 @@
+<html>
```

Comment:
Remove the test-output folder from your repo. You can ignore tracking for this folder by adding it to .gitIgnore